src/graph_1.py

```diff
@@ -64,7 +65,9 @@ def neighbors(self, val):
 
     def adjacent(self, val1, val2):
         """Return true if an edge connects "val1" and "val2"."""
-        if (val1, val2) in self.edges or (val2, val1) in self.edges:
+        if (val1, val2) in self.edges:
             return True
-        else:
+        elif val1 or val2 in self.node_list:
```

Comment:
If the `if` statement here isn't `True` then this `elif` will always evaluate to `True`